tests/integration/gov/genesis_test.go

```diff
@@ -204,23 +204,24 @@ func TestImportExportQueues_ErrorUnconsistentState(t *testing.T) {
 	suite := createTestSuite(t)
 	app := suite.app
 	ctx := app.BaseApp.NewContext(false)
-	require.Panics(t, func() {
-		gov.InitGenesis(ctx, suite.AccountKeeper, suite.BankKeeper, suite.GovKeeper, &v1.GenesisState{
-			Deposits: v1.Deposits{
-				{
-					ProposalId: 1234,
-					Depositor:  "me",
-					Amount: sdk.Coins{
-						sdk.NewCoin(
-							"stake",
-							sdkmath.NewInt(1234),
-						),
-					},
+
+	err := gov.InitGenesis(ctx, suite.AccountKeeper, suite.BankKeeper, suite.GovKeeper, &v1.GenesisState{
```

Comment:
deep dive on this, this use to check for panic, now we remove the panic and instead return an err here but the same panic behavior still persist. Should we just return to expecting panic here

Reply:
apparently we are passing genesisState with only deposits so it always panic with invalid memory address or nil pointer runtime err. What was the expecting err for this before ? and we should use `PanicsWithError` to assert that the code is panicking for the expected reason